drivers/SmartThings/zigbee-switch/src/init.lua

```diff
@@ -85,6 +89,40 @@ local device_init = function(self, device)
   end
 end
 
+local function device_added(driver, device, event)
+  device:set_find_child(find_child)
```

Comment:
I would move setting this function to only be set if at least 1 child device is created.